migrations/1506763059112-example-get-mapping.js

```diff
@@ -16,9 +16,6 @@ module.exports.up = function (next) {
     
     next()
   })
-
-    
-  next()
```

Comment:
Double call next() function break next file processing and mapping not set correctly.